main.nf

```diff
@@ -394,6 +396,9 @@ process markDuplicates {
         ASSUME_SORTED=true \\
         PROGRAM_RECORD_ID='null' \\
         VALIDATION_STRINGENCY=LENIENT
+
+    #Printing out version number to standard out
+    echo "File name: $bam_markduplicates MarkDuplicates version "$(java -Xmx2g -jar $PICARD_HOME/picard.jar  MarkDuplicates --version 2>&1)
```

Comment:
That it _is_ referencing `picard`. As `MarkDuplicates` is a submodule of `Picard`, I assume that this is printing the version of Picard. So it would be more accurate to say `MarkDuplicates version` instead of `Picard version`.

This is super minor stuff, I'm being OCD sorry.